src/main/java/emu/grasscutter/command/commands/GiveAllCommand.java

```diff
@@ -22,7 +22,8 @@ public final class GiveAllCommand implements CommandHandler {
     public void execute(Player sender, Player targetPlayer, List<String> args) {
         if (targetPlayer == null) {
             CommandHandler.sendMessage(sender, translate("commands.execution.need_target"));
-            return;
+//            return;
+            targetPlayer = sender;
```

Comment:
This is incorrect behavior as per new targeting system. You are also being inconsistent with both displaying error message AND continuing to execute.

Reply:
Give all command will detect if sender is valid, so don't mind if sender is valid.

Reply:
Please see https://github.com/Grasscutters/Grasscutter/pull/499. You are breaking behavior of the command system by this change and not even doing it properly, by both continuing execution AND displaying (now improper) error message.

Reply:
In addition, this should NOT be necessary - `CommandMap` should handle translation of current player to `targetPlayer` if no other target was provided automatically, without separate commands having to deal with it.

Reply:
This changes revert after deal with a bug of NPE, and it will be nothing happened after both of sender is null or empty.